pkg/controller/acmechallenges/controller.go

```diff
@@ -63,6 +64,7 @@ type Controller struct {
 
 	watchedInformers []cache.InformerSynced
 	queue            workqueue.RateLimitingInterface
+	metrics          *metrics.Metrics
```

Comment:
Each controller shouldn't need its own pointer to metrics, and if it does then I think they should all point to the same actual instantiation 😄

Reply:
I have removed them all to just call with the global `metrics.Default`, except for the certificates controller which already had a `metrics` pointer. However I'm not sure if this controller needs this anyway since this is just an alias for the same `metrics.Default`.
Additionally I don't quite understand the need for the Prometheus metrics to be exposed globally themselves in the `metrics.go` file since they can be accessed via the global `metrics.Default` anyway, unless this is a requirement for Prometheus.